src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,57 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+@Nested
+@DisplayName("String 테스트")
+public class StringTest {
+    @Test
+    void split() {
+        //요구사항1 : assertj의 contains, containsExactly 기능으로
+        String[] result = "1,2".split( ",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    void substring() {
+        String strText = "(1,2)";
+        String[] strArr = {"(", ")"};
+        String strRslt = "";
+
+        //입력받은 문자 안에 strArr의 문자와 같은 것이 있으면 substring으로 제외
+        for (int i = 0; i < strText.length(); i++) {
+            Character cTxt = strText.charAt(i);
+            for (int j =0; j < strArr.length; j++) {
+                if (strArr[j].equals(Character.toString(cTxt))) {
+                    strRslt = strText.substring(j);
+                }
+            }
+        }
```

Comment:
요구사항에서 `substring()`을 활용하라고 했으니, 다음처럼 간단하게 작성해도 될 것 같아요.

![image](https://user-images.githubusercontent.com/64854054/230243000-5c146172-0064-4d31-b2ff-3a392886cbb9.png)


```suggestion
        String strRslt= "(1,2)".substring(1, 4);
```